src/ansys/dpf/post/data_object.py

```diff
@@ -66,38 +67,61 @@ def as_data_frame(self, columns=None, **kwargs):
         >>> import pandas as pd
         >>> from ansys.dpf import post
         >>> from ansys.dpf.post import examples
-        >>> solution = post.load_solution(examples.multishells_rst, legacy=False)
-        >>> #stress = solution.stress()
-        >>> #stress.xx.plot_contour(show_edges=False)
+        >>> solution = post.load_simulation(examples.multishells_rst)
```

Comment:
```suggestion
        >>> simulation = post.load_simulation(examples.multishells_rst)
```
I'd go for simulation since we use the ``.load_simulation()`` method. I prefer simulation over solution.

Reply:
@GuillemBarroso Thx for the catch!